day-one/create-resource-from-template.mdx

```diff
@@ -1,6 +1,169 @@
+        </Step>
+    </Steps>
+
+Both methods will lead you to the **Create a new resource in the catalog** page, where you can select your template.
+
+## Configuring Your New Resource
+
+Once you're on the **Create a new resource in the catalog** page, you'll configure the details for your new service:
+
+<Steps>
+    <Step title="Project and Template Selection">
+        <Frame>
+            <img src="/images/day-one/create-resource-from-template-part-1.png" />
+        </Frame>
+        <p>You'll see two dropdown menus: **template** and **blueprint**.</p>
+        <p>Select the desired **template** from the dropdown. You can choose from templates created from either existing services or blueprints.</p>
+    </Step>
+    <Step title="Resource Details">
+        <Frame>
+            <img src="/images/day-one/create-resource-from-blueprint-part-2.png" />
+        </Frame>
+        <p>In the **Resource** section, provide the following information:</p>
+        <ul>
+            <li>**Name:** Give your new resource a descriptive name (e.g., "OrderProcessingService").</li>
+            <li>**Description:** Add a brief description for the resource.</li>
+            <li>**Catalog Properties:** Your platform team may have defined specific catalog properties to help organize and categorize resources. Fill in the required and any relevant optional properties. These might include things like "Tech," "Type," "Domain," and "Lifecycle."</li>
+        </ul>
+        <Note>
+            Learn more about how catalog properties help organize your resources in the [Catalog Properties](/day-zero/catalog-properties) page.
+        </Note>
+    </Step>
+    <Step title="Git Repository Settings">
+        <Frame>
+            <img src="/images/day-one/create-resource-from-blueprint-part-3.png" />
+        </Frame>
+        <p>By default, your new resource will be associated with the default Git repository configured for the project.</p>
+        <p>If you need to use a different repository:</p>
+        <ol>
+            <li>Click the **Override Default Settings** slider.</li>
+            <li>Select the desired Git organization.</li>
+            <li>Choose the specific repository you want to associate with this resource.</li>
+        </ol>
+
+        <Note>
+        When you create a new Amplication project you can choose to give it a default git repo.
+        That repo will be used for all new resources creating in that project, but it can you can override it.
+        </Note>
+
+    </Step>
+    <Step title="Create the Resource">
+        Once you've filled in all the necessary details, click the **Create Resource** button.
+    </Step>
+</Steps>
+
+## Exploring Your New Resource
+
+After creating your resource from a template, you'll be automatically taken to its resource overview page. The available tabs and settings will depend on whether the template was based on a service or a blueprint.
```

Comment:
I think we should write something like :

The functionality and content avilable for your new resource depends on the type of your specific resource. you may see some or all of the below

See [link] for more details about resources and blueprints